LICENSE.md

```diff
@@ -0,0 +1,23 @@
+Original dagre-d3 copyright: Copyright (c) 2013 Chris Pettitt
+Original dagre copyright: Copyright (c) 2012-2014 Chris Pettitt
+Original graphlib copyright: Copyright (c) 2012-2014 Chris Pettitt
+
+Copyright (c) 2022-2023 Thibaut Lassalle and dagre-es contributors
```

Comment:
I took this name from commit https://github.com/tbo47/dagre-es/commit/2af040cde688b524dcf7aca21aef61ddf8d4c8e5.

You're welcome to change this. You can also add other names if you want, but I've just done `and dagre-es contributors` to keep things simple.